apis/pipeline/pipeline.yaml

```diff
@@ -3658,112 +3637,6 @@ paths:
                 default:
                     $ref: '#/components/responses/Error'
 
-    /api/v1/orgs/{orgId}/clusters/{id}/features:
```

Comment:
I think we still call these from the CLI, I'm not sure about the web UI.

Reply:
:sadface: